dev-manuals/test-plans/keycloak/u-keycloak-01-standard-login.md

```diff
@@ -0,0 +1,36 @@
+# Keycloak: standard login (`u.keycloak-01`)
+
+
+Version: V1.0, 25.09.2022 \
+Author: Leon Layer
+
+## Description
+
+Standard login with user email and password.
+
+## Precondition
+
+No user is logged in. If the user was logged in, he has logged out so that he is no longer logged in via the cookies in the browser.
+
+## Postcondition
+
+The user is logged in and redirected to the start page where he can start or edit an "overworld" course.
+
+## Typical procedure
+
+1. Open the landing-page
+2. Go to login
+3. Fill in correct username or correct email and correct password 
+4. Choose sign in
+
+## Alternative procedures
+
```

Comment:
```suggestion

3.1. An incorrect username, email, or password is entered - The user is not redirected and is notified that their input was incorrect
```
By doing that, you can delete test `02` (don't forget to adapt the linkages)

What you can then do is create the `register` test I mentioned above as the new `02`.